packages/gatsby-transformer-remark/src/extend-node-type.js

```diff
@@ -599,10 +601,8 @@ module.exports = (
       tableOfContents: {
         type: `String`,
         args: {
-          pathToSlugField: {
-            type: `String`,
-            defaultValue: `fields.slug`,
-          },
+          relative: `Boolean`,
```

Comment:
I would go with second option here and use `absolute`, but set its default value to `true` for now (to avoid forcing breaking change) and keep default value for `pathToSlugField` as it is right now.

I agree with you that proposed behaviour is probably better default, but in my opinion this change is not big enough to warrant releasing new major version of this plugin, but we should track it and make the change for default values (make `absolute` default to false and `pathToSlugField` default to null/empty string) so we apply those changes when we really have something that would warrant major version bump.

Reply:
That make sense. Change as requested.